src/lib.rs

```diff
@@ -1,10 +1,80 @@
+//!# Rust Spice SDK
```

Comment:
If this is a duplicate of `README.md` then we will want to only have one source of truth. Can we generate the README from this comment doc?

Reply:
+1